libs/dmlf/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+#
+# F E T C H   D I S T R I B U T E D   M L  F R A M E W O R K   L I B R A R Y
+#
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+project(fetch-dmlf)
+
+# CMake Configuration
+include(${FETCH_ROOT_CMAKE_DIR}/BuildTools.cmake)
+
+# Compiler Configuration
+setup_compiler()
+
+# ------------------------------------------------------------------------------
+# Main Library Target
+# ------------------------------------------------------------------------------
+
+setup_library(fetch-dmlf)
+target_link_libraries(fetch-dmlf
+  INTERFACE fetch-core
+                             fetch-ml
+                             fetch-network
+                             fetch-muddle
+                             fetch-crypto
+                             fetch-math
+                             fetch-vm
+                             fetch-vm-modules
+                             pthread)
+
+# Test targets
+# add_test_target()
```

Comment:
commented out code. if you don't need it remove it.

Reply:
Test coming in later pr

Reply:
I had assumed that to be the case. That later pr can add these lines back in.
It's better to just never put in commented out code

Reply:
OK